llvm/test/tools/llvm-cxxfilt/no-params.test

```diff
@@ -0,0 +1,11 @@
+RUN: llvm-cxxfilt             _ZN1f2baC2ERKNS_2baIT_EE _ZN2baC2EOT0_ | FileCheck %s --check-prefix=CHECK-PARAMS
+RUN: llvm-cxxfilt          -p _ZN1f2baC2ERKNS_2baIT_EE _ZN2baC2EOT0_ | FileCheck %s --check-prefix=CHECK-NO-PARAMS
+RUN: llvm-cxxfilt --no-params _ZN1f2baC2ERKNS_2baIT_EE _ZN2baC2EOT0_ | FileCheck %s --check-prefix=CHECK-NO-PARAMS
+
+# f::ba::base object constructor(f::ba<> const&)
+CHECK-PARAMS: _ZN1f2baC2ERKNS_2baIT_EE
```

Comment:
It worth adding some explanation, as a comment, of why name hasn't been demangled in case of CHECK-PARAMS.